packages/block-library/src/image/index.php

```diff
@@ -82,7 +83,7 @@ function render_block_core_image( $attributes, $content ) {
 						$body_content
 						<div data-wp-body="" class="wp-lightbox-overlay"
 							data-wp-bind.role="selectors.core.image.roleAttribute"
-							aria-labelledby="$image_lightbox_id"
+							aria-label="$dialog_label"
```

Comment:
We should probably also escape the dialog label here.

Reply:
Only user provided texts are escaped in WordPress core.